packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/list/translations/Messages_fr_FR.json

```
@@ -10,7 +10,7 @@
   "octavia_load_balancer_list_l7_policies_actions_edit": "Editer",
   "octavia_load_balancer_list_l7_policies_actions_manage_rules": "Gérer les L7 Rules",
   "octavia_load_balancer_list_l7_policies_actions_delete": "Supprimer",
-  "octavia_load_balancer_create_l7_policy_success": "Votre L7 policy {{ policy }} a été créée avec succès. Vous pouvez à présent créer une <a href=\"{{ link }}\">L7 Rule</a>.",
+  "octavia_load_balancer_create_l7_policy_success": "Votre L7 policy {{ policy }} a été créée avec succès. Vous pouvez à présent créer une <a href=\"{{ link }}\" data-track-on=\"click\" data-track-type=\"action\" data-track-name=\" {{ trackingHit }} \">L7 Rule</a>.",
```

As `L7 Rule` is a common label for all languages, maybe you should transform your translation to avoid adding tracking here. For example :
```suggestion
  "octavia_load_balancer_create_l7_policy_success": "Votre L7 policy {{ policy }} a été créée avec succès. Vous pouvez à présent créer une {{L7RuleLink}}.",  
```

and add a parameter in the translate function 
```
{'L7RuleLink': `<a href="${link}" data-track-on="click" data-track-type="action" data-track-name="${trackingHit}">L7 Rule</a>`}
```
What do you think ?

---

It is not possible to use directive inside the Alerter, so for now this has been removed but we could look into a different solution at some point